package.json

```diff
@@ -24,6 +24,9 @@
         "@tailwindcss/forms": "^0.5.2",
         "@tailwindcss/line-clamp": "^0.4.0",
         "@tailwindcss/typography": "^0.5.0",
+        "@testing-library/dom": "^9.2.0",
+        "@testing-library/jest-dom": "^5.16.5",
+        "@testing-library/user-event": "^14.4.3",
```

Comment:
Industry-standard packages for querying, interacting with, and testing headless DOM environments in Jest and Vitest. It wouldn't surprise me if the Vue or React flavors are used in Laravel Livewire settings.